docs/user/dashboard/use-dashboards.asciidoc

```diff
@@ -0,0 +1,177 @@
+= Use and filter dashboards
+
+[float]
+[[search-or-filter-your-data]]
+== Search and filter your dashboard data
+
+{kib} supports several ways to explore the data displayed in a dashboard more in depth:
+
+* The **query bar**, using KQL expressions by default.
+* The **Time range**, that allows you to display data only for the period that you want to focus on. You can set a global time range for the entire dashboard, or specify a custom time range for each panel.
+* **Controls**, that dashboard creators can add to help viewers filter on specific values.
+* **Filter pills**, that you can add and combine by clicking on specific parts of the dashboard visualizations, or by defining conditions manually from the filter editor. The filter editor is a good alternative if you're not comfortable with using KQL expressions in the main query bar.
+* View the data of a panel and the requests used to build it.
+
+//You can combine the filters with any panel filter to display the data you want to see.
+
+This section shows the most common ways for you to filter dashboard data. For more information about {kib} and {es} filtering capabilities, refer to <<kibana-concepts-analysts,{kib} concepts>>.
+
+[float]
+=== Filter dashboards using the KQL query bar
+
+The query bar lets you build filters using <<kuery-query,{kib} Query Language (KQL)>>. When typing, it dynamically suggests matching fields, operators, and values to help you get the exact results that you want.
+
+image::images/dashboard-filter-kql.png[KQL filter dynamically suggesting values, width=40%]
+
+[float]
+=== Set a time range
+
+The data visible in a dashboard highly depends on the time range that is applied. In a dashboard, you can select a time range that applies globally to all panels, or set a custom time range for specific panels.
+
+[float]
+==== Apply a global time range to an entire dashboard
+
+The global time range menu is located right next to the query bar, in the dashboard's header. With this menu, you can select the time range to apply, and set the frequency for refreshing the dashboard data. Setting the time range is a common action in {kib}. Refer to <<set-time-filter>> for more details.
+
+image::images/dashboard-global-time-range.png[Time range menu with multiple time range suggestions, width=40%]
+
+[float]
+==== Apply a custom time range to a panel
+
+**To apply a panel-level time range:**
+
+. Open the panel menu, then select *Settings*.
+
+. Turn on *Apply a custom time range*.
+
+. Enter the time range you want to view, then *Apply* it.
+
+**To view and edit the time range applied to a specific panel:**
+
+When a custom filter is active for a single panel, it is indicated in the panel's header.
```

Comment:
how about changing "custom filter" to "custom time range" to avoid confusion with panel filters.

Reply:
🤦 I thought I did. Done now!